frontend/packages/kubevirt-plugin/src/components/vm.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from 'react';
+
+import {
+  getName,
+  getNamespace,
+  getUid,
+  // VmStatus,
+  // getSimpleVmStatus,
+  // VM_SIMPLE_STATUS_ALL,
+  // VM_SIMPLE_STATUS_TO_TEXT,
+  //  getResource,
+  //  DASHES,
+} from 'kubevirt-web-ui-components';
+
+import { NamespaceModel } from '@console/internal/models';
+import { ListHeader, ColHead, List, ListPage, ResourceRow } from '@console/internal/components/factory';
+import { ResourceLink } from '@console/internal/components/utils';
+// import { actions } from '../../module/okdk8s';
+
+import {
+  VirtualMachineModel,
+  // VirtualMachineInstanceModel,
+  // VirtualMachineInstanceMigrationModel,
+} from '../models';
+
+// import { openCreateVmWizard } from '../modals/create-vm-modal';
+// import { menuActions } from './menu-actions';
+// import { WithResources } from '../utils/withResources';
+
+const mainRowSize = 'col-lg-4 col-md-4 col-sm-6 col-xs-6';
+const otherRowSize = 'col-lg-4 col-md-4 hidden-sm hidden-xs';
+
+const VMHeader = props => <ListHeader>
+  <ColHead {...props} className={mainRowSize} sortField="metadata.name">Name</ColHead>
+  <ColHead {...props} className={otherRowSize} sortField="metadata.namespace">Namespace</ColHead>
+  <ColHead {...props} className={mainRowSize} sortField="spec.running">State</ColHead>
+</ListHeader>;
+
+const VMRow = ({ obj: vm }) => {
+  const name = getName(vm);
+  const namespace = getNamespace(vm);
+  const uid = getUid(vm);
+
+  /*
+  TODO(mlibra) To keep recent PR minimal, relations to other objects will be migrated as a next step.
+               Keeping the code here for reference.
+  const migrationResources = getResource(VirtualMachineInstanceMigrationModel, {namespace});
+  const resourceMap = {
+    pods: {
+      resource: getResource(PodModel, { namespace }),
+    },
+    migrations: {
+      resource: migrationResources,
+    },
+  };
+  */
+
+  return <ResourceRow obj={vm}>
+    <div className={mainRowSize}>
+      <ResourceLink kind={VirtualMachineModel.kind} name={name} namespace={namespace} title={uid} />
```

Comment:
I'd probably take `title={uid}`. I don't think it's particularly helpful. (I know we use it elsewhere.)